multi_livechat/models/mail_channel.py

```diff
@@ -21,16 +26,17 @@ def _prepare_multi_livechat_channel_vals(
     ):
         return {
             "channel_partner_ids": [(4, pid) for pid in partner_ids],
-            "public": "groups",
-            "group_public_id": self.env.ref("base.group_user").id,
+            # "public": "groups", # V16 dropout public field
+            "group_public_id": False,  # V16 checks contrains with group_public_id self.env.ref("base.group_user").id,
```

Comment:
Field `public` is removed in v16 -> ok

Field `group_public_id` is computed in v16 with `readonly=False`. Maybe just remove the field from  result?

https://github.com/odoo/odoo/blob/ef8a9cf01e67f518d85e30221a596372c358a65c/addons/mail/models/mail_channel.py#L196-L201

Reply:
I remove Field group_public_id, and it's working after I tested it.